docs/user_guide/plugin_development/messaging.rst

```diff
@@ -110,12 +106,12 @@ like so:
         @botcmd(template="hello")
         def hello(self, msg, args):
             """Say hello to someone"""
-            response = tenv().get_template('hello.html').render(name=args)
+            response = tenv().get_template('hello.md').render(name=args)
```

Comment:
on 3.1.x this is still true
but on 3.2 it will be send_templated (current head)